front/src/app/components/common/benchmark/index.tsx

```diff
@@ -0,0 +1,120 @@
+import * as React from 'react';
+import { IBenchmarkMap } from 'app/api/types';
+import { PropertyList, IPropertyListItem } from '../property-list';
+
+interface IBenchmarkProps {
+    className?: string;
+    keys: string[];
+    data: IBenchmarkMap;
+}
+
+interface IState {
+    keys: string[];
+    config: IPropertyListItem[];
+    data: IBenchmarkMap;
+}
+
+export class Benchmark extends React.PureComponent<IBenchmarkProps, IState> {
+    public state = {
+        config: Array.prototype,
+        keys: Array.prototype,
+        data: {} as IBenchmarkMap,
```

Comment:
empty object is not instance of IBenchmarkMap
as - very dangerous word. 
u can use ```Partial<IBenchmarkMap>``` in IState